includes/features/SU-SWS/stanford_person/stanford_person.feature

```diff
@@ -17,239 +17,13 @@ Feature: Stanford Person
-    Given I am on "people/department-administrator/grid"
-    Then I should see a ".views-row-first" element
-    And I should see a ".view-stanford-person-grid" element
-    And I should see the "People" heading in the "First sidebar" region
-
-  @content @deploy @stanford
-  Scenario: Profiles layout for department-administrator
-    Given I am on "people/department-administrator/profiles"
-    Then I should see a ".views-row-first" element
-    And I should see a ".view-stanford-person-profiles" element
-    And I should see the "People" heading in the "First sidebar" region
-
-  @content @deploy @stanford
-  Scenario: List layout for department-administrator
-    Given I am on "people/department-administrator/list"
-    Then I should see a ".views-row-first" element
-    And I should see a ".view-stanford-person-list" element
-    And I should see the "People" heading in the "First sidebar" region
-
-  @content @deploy @stanford
-  Scenario: Directory layout for department-administrator
-    Given I am on "people/department-administrator/directory"
-    Then I should see a ".views-row-first" element
-    And I should see a ".view-stanford-person-directory" element
-    And I should see the "People" heading in the "First sidebar" region
-
-  @content @deploy @stanford
-  Scenario: Grid layout for graduate-students
-    Given I am on "people/graduate-students/grid"
-    Then I should see a ".views-row-first" element
-    And I should see a ".view-stanford-person-grid" element
-    And I should see the "People" heading in the "First sidebar" region
-
-  @content @deploy @stanford
-  Scenario: Profiles layout for graduate-students
-    Given I am on "people/graduate-students/profiles"
-    Then I should see a ".views-row-first" element
-    And I should see a ".view-stanford-person-profiles" element
-    And I should see the "People" heading in the "First sidebar" region
-
-  @content @deploy @stanford
-  Scenario: List layout for graduate-students
-    Given I am on "people/graduate-students/list"
-    Then I should see a ".views-row-first" element
-    And I should see a ".view-stanford-person-list" element
-    And I should see the "People" heading in the "First sidebar" region
-
-  @content @deploy @stanford
-  Scenario: Directory layout for graduate-students
-    Given I am on "people/graduate-students/directory"
-    Then I should see a ".views-row-first" element
-    And I should see a ".view-stanford-person-directory" element
-    And I should see the "People" heading in the "First sidebar" region
-
-  @api @prod @dev @deploy @stanford @live
-  Scenario: Affiliates column on Manage Person
-    Given I am logged in as a user with the "administrator" role
-    When I go to "admin/manage/people"
-    Then I should see 1 or more ".views-field-field-s-person-affiliation" elements
+  @deploy
+  Scenario: Stanford Person CAP node page
+    Given I am on "people/jacob-smith"
+    Then I should see a ".postcard-left" element
```

Comment:
Why are we testing for .postcard-left? If we are testing to see if the person page exists the test for the url is fine. If you are testing to see if the content and other elements are available this test should be added to.